packages/lib/gpcircuits/circuits.json

```diff
@@ -28,6 +28,35 @@
       "globalWatermark"
     ]
   },
+  "proto-pod-gpc_1o-5e-6md-0x0l-0x0t": {
```

Comment:
Interesting realization: I hadn't thought about the fact that when the first dimension is zero the NxM format can be confused for hexadecimal.
I'm not suggesting changing it, just wondering if this will someday cause an unexpected bug in a parser which treats `0x0` as a single number.